src/models/model.cc

```diff
@@ -546,7 +546,7 @@ namespace ctranslate2 {
     }
 
     std::vector<std::shared_ptr<const Model>>
-    load_replicas(const std::string& model_path,
+    load_replicas(models::ModelReader& model_reader,
```

Comment:
As the error in the CI indicates, this function is also used in the Python wrapper. I suggest adding an overload taking the `std::string`, which calls the function taking the model reader.